docs/tutorials/solar_flares.ipynb

```diff
@@ -844,45 +1062,51 @@
    "cell_type": "markdown",
    "id": "61d012c3-76f6-41e8-9294-686bad746738",
    "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
     "tags": []
    },
    "source": [
-    "---\n",
-    "<details>\n",
-    "<summary class=\"solution\">\n",
-    "  Solution\n",
-    "</summary>\n",
-    "\n",
-    "<br>\n",
-    "Bin in time and keep the existing spatial binning.\n",
-    "`transpose` is used to put the time dimension on the slider in the plot.\n",
-    "\n",
-    "```python\n",
-    "temporal_and_spatial = spatial.bin(peak_time=200)\n",
-    "to_plot = temporal_and_spatial.hist()\n",
-    "to_plot.transpose(['peak_time', 'y', 'x']).plot(vmin=0.0 * sc.units.one,\n",
```

Comment:
You could keep this in, by using the `slicer` plot to show time slices of xy maps?
E.g. `pp.slicer(temporal_and_spatial.hist(), keep-['y', 'x'])`